From f4e6712865a3770082b6b315f5576c6db0bd16fa Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Mon, 07 Feb 2022 12:04:15 +0200
Subject: [PATCH] [SW-0] gitignore: add .vscode

Visual Studio Code creates a meta data directory (like Eclipse) in the
project directory.
Ignore this meta data directory.

Change-Id: I23c5506d7f28c591caceb83bd25fbf733162e688
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/.gitignore b/.gitignore
index 8c6bc44..452480d 100644
--- a/.gitignore
+++ b/.gitignore
@@ -1,3 +1,4 @@
 .cproject
 .project
 .settings/
+.vscode/
